xorgauth/accounts/oidc_provider_settings.py

```diff
@@ -13,18 +13,45 @@ def userinfo(claims, user):
     """
     claims['name'] = user.fullname
     claims['email'] = user.main_email
+    claims['study_year'] = user.study_year
+    claims['grad_year'] = user.grad_year
```

Comment:
I don't think `grad_year` and `study_year` should be added here, this function is for values that are defined in the general scopes.

Reply:
Indeed. I forgot to remove these lines (which in fact does no do anything as django-oidc-provider does not take into account unknown claims that would be set in ``userinfo()``.